ui/tests/acceptance/vault-reporting/index-test.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, currentURL, waitFor } from '@ember/test-helpers';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+const mockedEmptyResponse = {
+  data: {
+    auth_methods: {},
+    kvv1_secrets: 0,
+    kvv2_secrets: 0,
+    lease_count_quotas: {},
+    leases_by_auth_method: {},
+    replication_status: {},
+    secret_engines: {},
+  },
+};
+
+const mockedResponseWithData = {
+  data: {
+    auth_methods: { cats: 42 },
+    kvv1_secrets: 60,
+    kvv2_secrets: 40,
+    lease_count_quotas: {
+      global_lease_count_quota: { capacity: 420000, count: 210000, name: 'default' },
+      total_lease_count_quotas: 1,
+    },
+    namespaces: 1,
+    replication_status: {
+      dr_primary: true,
+      dr_state: 'enabled',
+      pr_primary: false,
+      pr_state: 'enabled',
+    },
+    secret_engines: { dogs: 43 },
+  },
+};
+
+module('Acceptance | vault-reporting', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    await login();
+  });
+
+  test('it visits the usage reporting dashboard and renders the header', async function (assert) {
+    await visit('/vault/usage-reporting');
+    assert.strictEqual(currentURL(), '/vault/usage-reporting', 'navigates to usage reporting dashboard');
+    assert.dom('.hds-page-header').includesText('Vault Usage', 'renders the "Vault Usage" header');
+  });
+
+  test('it renders the counters dashboard block with all expected counters', async function (assert) {
+    this.server.get('http://localhost:7357/v1/sys/utilization-report', () => mockedResponseWithData);
+    await visit('/vault/usage-reporting');
+    await waitFor('[data-test-dashboard-counters]');
```

Comment:
Up to you guys, but I recommend creating a vault-usage helper directory & file. Similar to something like [this](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/kubernetes/kubernetes-selectors.ts).